packages/dx-react-grid/src/types/data-type-provider/data-type-provider.types.ts

```diff
@@ -22,6 +22,8 @@ export namespace DataTypeProvider {
     value: any;
     /** Handles value changes. */
     onValueChange: (newValue: any) => void;
+    /** Specifies whether editing a column is enabled. */
+    editingEnabled: boolean;
```

Comment:
Should we make this property public?

Reply:
It is public :)

Reply:
I mean - should we add this property into our documentation?

Reply:
Yes, I'll add it